### ISSUE TITLE
Articulate Value Proposition

### ISSUE THREAD

**Issue by iros** [moebiolabs/moebio_framework#98]
![image](https://cloud.githubusercontent.com/assets/18690/9092028/9da792b2-3b71-11e5-8a09-d61c973fca4c.png)

**Comment by moebio** [comment]
@tafsiri @vlandham happy to see feedback on this. Do you think it can be improved, also by introducing some of the ideas of your version?

**Comment by vlandham** [comment]
indeed. I think we are going to work some of this into the website tomorrow, and come up with a composite version.

**Comment by moebio** [comment]
excellent!

**Comment by tafsiri** [comment]
@moebio @vlandham  I've taken a stab at incorporating some of this here https://github.com/moebiolabs/moebio_framework_site/pull/16

But I'll also try and respond to some of this here.

First off, thanks that was really helpful to see. Content wise we may not be able to incorporate all of this on the about page or intro blurb, but it gives me some ideas for how to direct producing more content. But i'll talk more about that below.

• analysis and visualization
• grammar of data types, transformation and combination
• awareness of the type and structure of the data object

I've updated the copy in the PR above to try and incorporate these points in a short way. 

• formal freedom

I think i've updated the content to say that graphics is an _additional_ thing that the fw provides. On the issue of formal freedom i'm not sure if I can think of a way to convey that. This could be because in some ways I think all tools provide some constraints notions of ways of making (in a manner similar to the way language constrains ways of thinking), and that this is not necessarily a bad thing. Now i think the thing that maybe you want to convey is that the space is not heavily constrained. But i think the showreel (and future examples) may be the best way to show that (rather than describe it). 

• canvas

I think its useful to state what kind of graphics api the fw provides, as that is the default people will use and what the materials around the fw will primarily rely on. We've tried to describe that concisely in the features area.

• non 1-1 approach

This is an interesting one, but somewhat hard to articulate. As I was watching the visualized video, i was thinking this might be a good "show rather than tell", candidate. And what i mean by that is, that for a number of these points we could demonstrate this with a blog post that builds out (or deconstructs) a visual example, e.g. the spiral timeline. I think a number of these bullet points could be feeders for blog posts/articles/examples that could be added to the site over time. They would elucidate the philosophical approach visually, making it more concrete.

• networks
+1 I think demonstrated well in the examples

• data science
+1 i think we've tried to hint to that as well in the about and the tutorial jim wrote.

• not imposing coding logic, combines with other frameworks
Yeah on this one i don't think it can be said without comparison, which i don't think is useful. As you said every framework provides a logic, and i agree with that. I think this one would be one we want to keep an eye on and see how others feel about this. I think this is the kind of thing that would emerge from the community response and inform our assumptions, but is hard to just state from the perspective of the author.
